### DISCUSSION TITLE
Include LICENSE.md?

### DISCUSSION THREAD

**Discussion by lee-jongwoo** [msk-apps/msk-markdown#19] [Ideas]
In our README.md, it says "MIT License", but we don't have a LICENSE.md.

[https://github.com/msk-apps/msk-markdown/community/license/new](https://github.com/msk-apps/msk-markdown/community/license/new)
Probably better to have a dedicated LICENSE.md for this repo.

Also, we'll need some more stuff like PR Templates and Contributing Guidelines.

**Comment by lee-jongwoo** [comment]
@kevinmmansour  Should the copyright holder name be `Kevin M. Mansour`, `MSK`, or something else?
Wasn't sure about that so I couldn't create the file.

**Reply by kevinmahrous** [reply]
Thanks! It's `MSK` since this's the company name. I made the license file today. I'll do PR Templates and Contributing Guidelines and maybe update IssuesTemplates since I have some homeworks to do.

**Comment by kevinmahrous** [comment]
OK. I have updated Issue templates as well as created MIT License file and created PR templates. I have also written the Contributing Guidelines but I would need to take a confirmation that's OK (for example, if there's any grammer mistakes or something that I wasn't clear about or something that need updates), if it's OK. I'll publish it.

```
# How to contribute?

First of all, thank you for taking the time to contribute to this project. We've tried to make a stable project and try to fix bugs and add new features continuously. You can help us do more.

# How to create a good issues?

Issues are usually bug reports, feature requests or documentation improvements.

Please, first, before submitting your issue, please check if there's an [existing one](https://github.com/msk-apps/msk-markdown/issues). 

If there's an old *same* issue but it's closed, feel free to create a new one but review first why the old issue got closed, in order to don't get your new issue also closed.

If there's no existing issues, feel free to add it! But we highly recommend to use GitHub issues templates that we made to write your issue rather than creating a blank one.

Issues should contain clear and concise description of the issue, screenshot and desktop or smartphone information, if applicable.

Don't ping or add comments in order to bring attention to your issue. Please be patient until a contributor or maintainer decide to work on this issue. 

**Spam issues aren't allowed for any reason.**

# How to create a good pull request?

Please, first, before creating a new feature pull request, please create an issue or Github discussion in order to discuss the new feature with the community, contributors and maintainers to see if the new feature is applicable for implementing or not.

If the pull request is going to fix an existing issue that it's *already discussed*, so feel free to do!

We recommend to use PR templates that we made rather than creating a blank one.

Pull requests should contain the number of the issue that fix or the new feature that'll be implemented, like so:

`#123`, `123` is the issue number.

And a clear and concise description of what this pull request does or fix.

Don't ping or add add comments in order to bring attention to your pul request. Please be patient until a maintainer review and decide to merge the pull request.

We recommend that pull requests push to the [`dev`](https://github.com/msk-apps/msk-markdown/tree/dev) branch and when there's one or two ready pull request, they'll be merged to `master` branch and deployed online (automatically after merge).

Because pushing to `master` branch will automatically deploy and because there might be other contributers that also work, so there might be conflict for their code, so deploying to `dev` is better.

**Spam pull requests aren't allowed for any reason.**

# How to create useful GitHub discussions?

GitHub discussions will be used for announcements about the project, but they might be used for asking questions about the project under the "Q&A" category. 

As well as if you have an idea and want to share. Feel free to do under the "Idea" category! 

If there's any general stuff, so it should go under the "General" category.

All GitHub discussions should follow [GitHub Community Guidelines](https://docs.github.com/articles/github-community-guidelines) and **Spam discussions aren't allowed for any reason.**
```

**Reply by lee-jongwoo** [reply]
I'm not an expert in English, but here's some improvements:
[Contributing Guidelines.docx](https://github.com/msk-apps/msk-markdown/files/7281821/Contributing.Guidelines.docx)

# How to contribute?

First of all, thank you for contributing to this project. We are trying to make this a stable project by fixing bugs and adding new features continuously. You can help us do more.

# How to create a good issue?

Issues are usually bug reports, feature requests, or documentation improvements.

 Please check if there is already a [same issue](https://github.com/msk-apps/msk-markdown/issues) before submitting yours.

If there is an old *same* issue that is closed, feel free to create a new one. Make sure to review why the original issue got closed to avoid your issue getting closed as well.

If there are no existing issues, it is absolutely okay to open one! But we highly recommend using the GitHub issue templates we made to describe your issue.

Issues should contain a clear and concise description of the issue, a screenshot of the problem, and device information, if applicable.

Don't ping or add comments to bring attention to your issue. Please be patient until a contributor or maintainer decides to work on your issue.

**Spam issues aren't allowed for any reason.**

# How to create a good pull request?

Please create an issue or Github discussion before opening a pull request to discuss the new feature with the community, contributors, and maintainers to see if the feature is applicable for implementation or not.

If the pull request fixes an issue that is *already discussed*, feel free to open it!

We recommend using the PR templates that we made instead of creating a blank one.

If the pull request is related to an issue, please mention the issue using the following format:

```
Fixes #123
Resolves #123

(123 is the issue number)
```

And also a clear and concise description of what the pull request does or fixes.

Don't ping or add additional comments to bring attention to your pull request. Please be patient until a maintainer reviews and decides to merge the pull request.

We recommend that pull requests push to the [`dev`](https://github.com/msk-apps/msk-markdown/tree/dev) branch. When there are multiple pull requests ready for merge, they will be merged to the `master` branch and will be deployed online automatically.

Please push to the `dev` branch to avoid merge conflicts with other PRs and automatic deploys.

**Spam pull requests aren't allowed for any reason.**

# How to create useful GitHub discussions?

GitHub discussions are used for announcements and questions about the project(under the "Q&A" category).

If you have a good idea, feel free to share it under the "Idea" category! 

All GitHub discussions should follow [GitHub Community Guidelines](https://docs.github.com/articles/github-community-guidelines).
**Spam discussions aren't allowed for any reason.# How to contribute?

First of all, thank you for contributing to this project. We are trying to make this a stable project by fixing bugs and adding new features continuously. You can help us do more.

# How to create a good issue?

Issues are usually bug reports, feature requests, or documentation improvements.

 Please check if there is already a [same issue](https://github.com/msk-apps/msk-markdown/issues) before submitting yours.

If there is an old *same* issue that is closed, feel free to create a new one. Make sure to review why the original issue got closed to avoid your issue getting closed as well.

If there are no existing issues, it is absolutely okay to open one! But we highly recommend using the GitHub issue templates we made to describe your issue.

Issues should contain a clear and concise description of the issue, a screenshot of the problem, and device information, if applicable.

Don't ping or add comments to bring attention to your issue. Please be patient until a contributor or maintainer decides to work on your issue.

**Spam issues aren't allowed for any reason.**

# How to create a good pull request?

Please create an issue or Github discussion before opening a pull request to discuss the new feature with the community, contributors, and maintainers to see if the feature is applicable for implementation or not.

If the pull request fixes an issue that is *already discussed*, feel free to open it!

We recommend using the PR templates that we made instead of creating a blank one.

If the pull request is related to an issue, please mention the issue using the following format:

```
Fixes #123
Resolves #123

(123 is the issue number)
```

And also a clear and concise description of what the pull request does or fixes.

Don't ping or add additional comments to bring attention to your pull request. Please be patient until a maintainer reviews and decides to merge the pull request.

We recommend that pull requests push to the [`dev`](https://github.com/msk-apps/msk-markdown/tree/dev) branch. When there are multiple pull requests ready for merge, they will be merged to the `master` branch and will be deployed online automatically.

Please push to the `dev` branch to avoid merge conflicts with other PRs and automatic deploys.

**Spam pull requests aren't allowed for any reason.**

# How to create useful GitHub discussions?

GitHub discussions are used for announcements and questions about the project(under the "Q&A" category).

If you have a good idea, feel free to share it under the "Idea" category! 

All GitHub discussions should follow [GitHub Community Guidelines](https://docs.github.com/articles/github-community-guidelines).
**Spam discussions aren't allowed for any reason.**

**Reply by kevinmahrous** [reply]
OK. I'll use that.

**Comment by kevinmahrous** [comment]
I've updated Issues Templates. I've created LICENSE file, PR Template as well as I finished Contributing Guidelines. Have I missed anything? If not, should we delete this discussion or keep it?

**Reply by lee-jongwoo** [reply]
GitHub has this [checklist](https://github.com/msk-apps/msk-markdown/community) of recommended community standards. But I’m not sure if we’ll *really* need stuff like code of conduct.
It’s not a really big project we have here, so will we really need that?
